tests/integration_tests/test_pure_tf_model.py

```diff
@@ -88,8 +88,10 @@ def test_pure_tf_model(csv_filename, tmpdir):
         skip_save_processed_input=True,
     )
 
+    # TODO(ksbrar) 
+    # print(input_features[0])
     ludwig_tf_model = ludwig_model.create_inference_graph()
-    print(ludwig_tf_model)
+    # print(ludwig_tf_model)
```

Comment:
Let's expand this test. We need to at least cover the following:

- Test different tokenizers
- Test handling invalid tokenizer
- Test equivalence of predictions from Python and TF models